server/www/index.html

```diff
@@ -10,7 +10,7 @@
         <link rel="stylesheet" href="/css/rtl-style.css" ng-disabled="! rtlEnabled">
         <link rel="stylesheet" href="/css/vendor.css">
 
-        <title>{{pageTitle}}{{siteTitle}}</title>
+        <title ng-bind-template="{{pageTitle}} {{siteTitle}}"></title>
```

Comment:
Do you think it'd make sense to add some default text here though? ie. `<title ng-bind-template="{{pageTitle}} {{siteTitle}}">Ushahidi</title>`